node/src/GlideClusterClient.ts

```diff
@@ -487,12 +493,16 @@ export class GlideClusterClient extends BaseClient {
      * console.log(result); // Output: {'addr': 'Connection Name', 'addr2': 'Connection Name', 'addr3': 'Connection Name'}
      * ```
      */
-    public async clientGetName(
-        route?: Routes,
-    ): Promise<ClusterResponse<string | null>> {
-        return this.createWritePromise<ClusterResponse<string | null>>(
+    public async clientGetName(options?: {
+        route?: Routes;
+        decoder?: Decoder;
+    }): Promise<ClusterResponse<GlideString | null>> {
+        return this.createWritePromise<ClusterResponse<GlideString | null>>(
             createClientGetName(),
-            { route: toProtobufRoute(route) },
+            {
+                route: toProtobufRoute(options?.route),
+                decoder: options?.decoder,
```

Comment:
nit: 
```
...options
```
should accomplish the same thing

Reply:
unfortunately, not, because both have `route` but in different type